### DISCUSSION TITLE
2023.8.0

### DISCUSSION THREAD

**Discussion by AndrewBastin** [hoppscotch/hoppscotch#3308] [Announcements]
[Read our release announcement here.](https://dev.to/hoppscotch/introducing-hoppscotch-enterprise-edition-and-a-plethora-of-features-the-hoppscotch-august-2023-release-you-cant-miss-hcg)

## What's Changed
* feat: introducing i18n support to admin dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/3051
* feat: placeholder component in hoppscotch-ui by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3123
* refactor: move hoppscotch-common tests to vitest by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3154
* feat: auto-complete recent history entries in URL bar by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3141
* feat: revamped spotlight by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3171
* chore: update generateSitemap usage by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3182
* redesigned search button by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3187
* docs: fixed shortcut keys for spotlight and shortcuts menu by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3192
* fix: firefox browser scrollbar issue by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3201
* fix: elastic overscroll on safari by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3221
* feat: context menu by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3180
* chore: type and UX improvement for SmartTree by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3126
* feat : Allow admins to revoke a team invite (HBE-230) by @ankitsridhar16 in https://github.com/hoppscotch/hoppscotch/pull/3162
* refactor: fonts are now bundled with packages by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3227
* fix: disable line wrapping in EnvInput component by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3230
* feat: introducing a new smart input hoppscotch ui component by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/3089
* feat: tippy menu for history and tab by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3220
* HBE-248 feat: prefix VITE_ added in conditional auth provider env variable  by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/3246
* spotlight: navigation searcher added by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3245
* refactor: revoke team invitation in admin dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/3232
* feat: environment quick peek by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3119
* Polish Environment Selector by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3260
* feat: introducing ability to conditionally enable auth providers in dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/3225
* spotlight: settings based actions added by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3244
* feat: inspections by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3213
* feat: spotlight collection searcher by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3262
* feat: load allowed login methods from .env by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3264
* fix: placeholder size and text overflow on tab head by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3261
* feat: smart-tree component added to hoppscotch-ui by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3210
* refactor: move from network strategies to generic interceptor service by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3242
* chore: bump dependencies  by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3258
* feat: add spotlight definition by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3255
* feat: gql revamp by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/2644
* refactor: autofocus can be disabled in smart input hopp ui component by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/3273
* chore: minor ui improvements by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3274
* chore: removed unnecessary dependencies from hoppscotch-ui by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3077
* fix: vertical scroll not working on codemirror instance when lines are not wrapped by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3276
* fix: fix issues in displaying the suggestions menu on EnvInput by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3280
* feat: container registry friendly docker images and all-in-one container by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3193
* fix: autocomplete bug by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3285
* feat: cleaner save context handling for graphql by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3282
* fix: icons inside tooltip by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3283
* fix: tab right click rename bug by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3286
* HBE-258 hotfix: skip parameter in findMany in shortcode module by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/3294
* feat: move crisp out of common by @AndrewBastin in https://github.com/hoppscotch/hoppscotch/pull/3287
* fix: fix issues with codegen by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/3293
* fix: inspections bugs by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/3277
* fix: graphql operation highlight issue on focus changed by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3297
* chore: improved command labels and icons by @liyasthomas in https://github.com/hoppscotch/hoppscotch/pull/3295
* fix: spotlight actions on graphql by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/3299
* style: updated font size and truncation on fields in the invited users table in admin dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/3300

**Full Changelog**: https://github.com/hoppscotch/hoppscotch/compare/2023.4.8...2023.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/hoppscotch/hoppscotch/releases/tag/2023.8.0'>2023.8.0</a>.</em>

**Comment by ThraaxSession** [comment]
Will there be a docker image this time?

**Reply by liyasthomas** [reply]
Hi @ThraaxSession, Yes! We're now providing Docker images to our community and enterprise editions.
Learn more on  [Self-Hosting documentation](https://docs.hoppscotch.io/documentation/self-host/community-edition/install-and-build#docker).

**Reply by ThraaxSession** [reply]
Last times there weren't for months no docker images. Sorry for asking.

**Reply by jobartim44** [reply]
Great, thanks for this
